paymentsheet/src/main/java/com/stripe/android/link/LinkScreenContent.kt

```diff
@@ -58,7 +62,6 @@ internal fun LinkScreenContent(
             changeEmail = viewModel::changeEmail,
             onNavBackStackEntryChanged = viewModel::onNavEntryChanged,
             navigationChannel = viewModel.navigationFlow,
-            appearance = viewModel.linkConfiguration.linkAppearance
```

Comment:
👀 No longer need to drill `linkAppearance` down

Reply:
this is great!